CosmosContracts/juno-tools
# refactor(pages): remove interstitial airdrop pages

## Description

This PR updates or removes interstitial airdrop pages.

## Changes

- [x] make `/airdrops` as `/airdrops/list`
- [x] make `/airdrops/manage` as `/airdrops/create`

Note: to ensure we're not throwing dead links, previous interstitial pages will show corresponding pages.

## Screenshots

\-

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- make sure `/airdrops` is equal to `/airdrops/list`
- make sure `/airdrops/manage` is equal to `/aidrops/create`

## Links

\-
